sanowret-crystal.lic

```diff
@@ -0,0 +1,39 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
```

Comment:
We've generally been putting a link to elanthipedia in the header block and documenting things there.